code/addons/docs/package.json

```diff
@@ -121,8 +121,9 @@
     "ts-dedent": "^2.0.0"
   },
   "devDependencies": {
-    "react": "^16.14.0",
-    "react-dom": "^16.8.0",
+    "@types/react-dom": "^18.0.9",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
@thebuilder Could we upgrade the API without actually upgrading to react 18 ourselves?

Could we stick with react 16 for now for development?
I assume this change is actually backward compatible?

I'm worried this might be too risky of a change for 7.0 which is already in beta, and we want to not do too many disruptive changes at this point in time... 
But we can be convinced of course.

Reply:
It should already do that - It's just a `devDependecy`, so the addon uses the installed version of React. 
Using the new `createRoot` if it's React 18, and falling back to `render` for older versions.

Reply:
So yes, we can downgrade the version of react for dev - but not sure what would be gained from this? The build is passing with the new version.